app/models/customer.rb

```diff
@@ -20,6 +20,11 @@ def self.top_five_customers(merchant_id)
   end
 
   def self.top_five_customers_admin
-    joins(:transactions).where('transactions.result = 0').group(:last_name, :first_name).order(count: :desc).limit(5).count
+    joins(:transactions)
+      .where('transactions.result = 0')
+      .group(:last_name, :first_name)
+      .order(count: :desc)
+      .limit(5)
+      .count
```

Comment:
I am pretty sure this is not going to give us what you want... I am also not sure about this, but I have had more luck with using the aggregate in my select statement and providing an alias (that way you can use it later in the view).  Like this: `.select('invoice_items.*, sum('invoice_items.quantity * invoice_items.unit_price) as revenue')` and I also think you should include ':first_name, :last_name, in your select statement but then `.group(:id)` and `.order('revenue')`

Reply:
Oh wait, I was using the user story from your PR request... This is a different method. I see it below haha. Ignore most of that - but I would still think about a select statement here that might do more of what you want it to.